pagopa/io-app
# chore: Add auto commenting and uncommenting script for lollipop checks

## Short description
This PR adds a script to comment/uncomment the lollipop checks inside `startup.ts` that don't allow you to run the application on an Android emulator.

## List of changes proposed in this pull request
- Added comments inside `startup.ts` to facilitate spotting the incriminated code blocks.
- Added the shell script that automatically comment/uncomment the aforementioned code blocks.
- Added in `package.json` two scripts to run the above mentioned shell script.
  - `yarn lollipop_checks:comment`
  - `yarn lollipop_checks:uncomment`

## How to test
Try running `yarn lollipop_checks:comment`: you should see the lollipop checks inside `startup.ts` both commented.
Try running `yarn lollipop_checks:uncomment`: you should see no modifications inside `startup.ts` — all comments have been reverted.
